### PR TITLE
For review

### PR DESCRIPTION
**Pull request by rafimarc101**

### CODE REVIEW

File: src/components/SupplierManagementComponents/SupplierDetails.vue [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<template>
+    <b-container>
+        <b-row class="form">
+            <b-col sm="12">
+                <label class="text" for="input-small">Material Name</label>
```
Comment (epsotto) [inline src/components/SupplierManagementComponents/SupplierDetails.vue]: Please revisit the forms. The "for" element in the label denotes the "Id" element of the input. Add unique Ids for the inputs then apply them into the For in the labels.

File: src/components/SupplierManagementComponents/SupplierForm.vue [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<template>
+    <b-container>
+        <b-row class="form">
+            <b-col sm="12">
+                <label class="text" for="input-small">Supplier Name</label>
```
Comment (epsotto) [inline src/components/SupplierManagementComponents/SupplierForm.vue]: Please revisit the forms. The "for" element in the label denotes the "Id" element of the input. Add unique Ids for the inputs then apply them into the For in the labels.

File: src/components/SupplierManagementComponents/SupplierList.vue [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+            selectable
+            select-mode="single"
+            selectedVariant="success"
+            :items="items" 
+            show-empty
+            @row-selected="rowSelected" 
+            :fields="fields">
+        </b-table> 
+
+       {{selected}}
```
Comment (epsotto) [inline src/components/SupplierManagementComponents/SupplierList.vue]: Remove this if done testing table functionality.

File: src/components/ItemRequestComponents/QuotationDetails.vue [hunk 4]
```diff
@@ -1,38 +1,27 @@
...
-        <span mdb-icon icon="print"></span></a></p>
-      </b-col>
-    </b-row>
-        
-        
     <b-row class="form-1">
       <b-col sm="6">
-        <label id="text-1" for="input-small">Title:</label>
-        <b-form-input id="input-small" size="sm"></b-form-input>
+        <label class="text-1" for="input-small">Title:</label>
```
Comment (epsotto) [inline src/components/ItemRequestComponents/QuotationDetails.vue]: Please revisit the forms. The "for" element in the label denotes the "Id" element of the input. Add unique Ids for the inputs then apply them into the For in the labels.

File: src/components/ItemRequestComponents/GeneralItemRequestComponent.vue [hunk 5]
```diff
@@ -1,29 +1,28 @@
 <template>
-  <b-container fluid>
-    <h7>FILTERS</h7>
+  <b-container fluid>    
     <b-row class="form-1">
       <b-col sm="6">
-        <label id="text-1" for="input-small">Item Request ID:</label>
-        <b-form-input id="input-small" size="sm"></b-form-input>
+        <label class="text-1" for="input-small">Item Request ID:</label>
```
Comment (epsotto) [inline src/components/ItemRequestComponents/GeneralItemRequestComponent.vue]: Please revisit the forms. The "for" element in the label denotes the "Id" element of the input. Add unique Ids for the inputs then apply them into the For in the labels.

File: src/components/SupplierManagementComponents/SupplierDetails.vue [hunk 6]
```diff
@@ -23,7 +23,7 @@
 </script>
 
 <style scoped>
-.input-small{
+.input-small{np
```
Comment (epsotto) [inline src/components/SupplierManagementComponents/SupplierDetails.vue]: Is "np" supposed to be here?

File: src/components/ItemRequestComponents/GeneralItemRequestComponent.vue [hunk 7]
```diff
@@ -14,11 +14,11 @@
     <b-row class="form-2">
       <b-col sm="4">
         <label class="text-1" for="input-small">Date From:</label>
```
Comment (epsotto) [inline src/components/ItemRequestComponents/GeneralItemRequestComponent.vue]: Please revisit the forms. The "for" element in the label denotes the "Id" element of the input. Add unique Ids for the inputs then apply them into the For in the labels.

File: src/components/ItemRequestComponents/ItemStockBalance.vue [hunk 8]
```diff
@@ -12,8 +11,8 @@
             @row-selected="rowSelected" 
             :fields="fields">
 
-            <template slot="thead-top" slot-scope="data">
-        <tr>
+        <template slot="thead-top" slot-scope="data">
```
[the remark this answers is not in the document]
Comment (rafimarc101) [reply src/components/ItemRequestComponents/ItemStockBalance.vue]: Template is for table head for multiple column.. should i remove it?

File: src/components/ItemRequestComponents/QuotationLists.vue [hunk 9]
```diff
@@ -10,7 +9,7 @@
             :items="items" 
             show-empty
             @row-selected="rowSelected" 
-            :fields="fields">
+            :fields="fields">            
         </b-table> 
 
        {{selected}}
```
[the remark this answers is not in the document]
Comment (rafimarc101) [reply src/components/ItemRequestComponents/QuotationLists.vue]: Already removed...tia!

File: src/components/SupplierManagementComponents/SupplierDetails.vue (hunk 1 above)
Comment (rafimarc101) [reply src/components/SupplierManagementComponents/SupplierDetails.vue]: <label class="text" for="item-name">Material Name</label>

is this right?

File: src/components/SupplierManagementComponents/SupplierDetails.vue (hunk 6 above)
Comment (rafimarc101) [reply src/components/SupplierManagementComponents/SupplierDetails.vue]: where did that came from?

**Pull request merged**